### DISCUSSION TITLE
Connecting to ECU

### DISCUSSION THREAD

**Discussion by addyhash** [agronick/aa-torque#108] [Q&A]
Hi gents.
I cannot seem to get live data on aatorque. I have torque pro working on my phone with live data but as torque dash just says connecting to ECU.  Would it bc my torque pro is side loaded ? And aatorque can't see it ?

**Comment by agronick** [comment]
Sounds like you are running an old version of Torque.

**Reply by orogvany** [reply]
seeing this with 1.12.101, which google says is latest?
